packages/react/src/patterns/blocks/LeadSpaceBlock/README.md

```diff
@@ -0,0 +1,149 @@
+# Lead Space Block
+
+> The "Lead Space Block" pattern leverages the Content Block as the
+> container of this pattern, and allows to select media with link lists 
+> followed by a button.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+##### CSS
+
+```css
+
+@import '@carbon/ibmdotcom-styles/scss/patterns/blocks/leadspace-block/leadspace-block';
+```
+
+##### JS
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { LeadSpaceBlock } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  const title = 'Continuous delivery';
+  
+      const heading= text(
```

Comment:
The README shouldn't be using the storybook knobs at all, looks like you're including `text`, `object`, etc.

Reply:
@jeffchew yeah, I wiil correct those.

Reply:
I've changed them.